test/extensions/geoip_providers/maxmind/test_data/BUILD

```diff
@@ -0,0 +1,13 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+filegroup(
+    name = "geolocation_databases",
+    srcs = glob(["*.mmdb"]),
+)
```

Comment:
Checked-in mysterious undocumented binary blobs are pretty unfriendly for test validation - is there any chance there's a convenient interface for generating these files from something human-readable? If so you could use a genrule or set of genrules with outs=[the mmdb files] to get the same benefits without the opacity.

(If there is no easy way to generate the binary files then I guess this has to be acceptable.)

Reply:
@ravenblackx Maxmind maintainers answered the question here: https://github.com/maxmind/MaxMind-DB/issues/121#issuecomment-1741392925. I will need to investigate how to use the Go tool and bazel genrule.

Reply:
I would say pulling in another repo dependency for this (that doesn't have bazel bindings) is probably too high a cost for the benefit, such that just some comments about what the binary mmdb files contain is probably the more reasonable approach.